### PR TITLE
[System.Windows.Forms] Proper padding and border width for DataGridViewTextBoxCell. Fixes #60514

### PR DESCRIPTION
**Pull request by YuriyGS**
Proper use padding and border width for DataGridViewTextBoxCell.

### CODE REVIEW

**Comment by akoeplinger** [comment]
@monojenkins whitelist

File: mcs/class/System.Windows.Forms/System.Windows.Forms/DataGridViewTextBoxCell.cs [hunk 1]
```diff
@@ -211,14 +211,19 @@ protected override void Paint (Graphics graphics, Rectangle clipBounds, Rectangl
...
 
 				Rectangle contentbounds = cellBounds;
-				
-				contentbounds.Height -= 2;
-				contentbounds.Width -= 2;
-				
-				// If we are top aligned, give ourselves some padding from the top
-				if (((int)cellStyle.Alignment & 7) > 0) {
-					contentbounds.Offset (0, 2);
-					contentbounds.Height -= 2;
```
Comment (akoeplinger) [inline mcs/class/System.Windows.Forms/System.Windows.Forms/DataGridViewTextBoxCell.cs]: can you explain why we don't need this anymore in the new code?

Comment (alexischr) [reply mcs/class/System.Windows.Forms/System.Windows.Forms/DataGridViewTextBoxCell.cs]: @akoeplinger it looks like that code was a workaround to pad/offset with some "good enough" values when not processing advancedBorderStyle and cellStyle.Padding

Comment (knocte) [reply mcs/class/System.Windows.Forms/System.Windows.Forms/DataGridViewTextBoxCell.cs]: ideally this reasoning should go in the commit message, not in the PR

**Pull request merged**